ports/geckolib/glue.rs

```diff
@@ -3373,7 +3373,7 @@ pub extern "C" fn Servo_DeclarationBlock_SetTextDecorationColorOverride(
     use style::properties::PropertyDeclaration;
     use style::properties::longhands::text_decoration_line;
 
-    let mut decoration = text_decoration_line::computed_value::none;
+    let mut decoration = text_decoration_line::get_initial_value();
```

Comment:
Again, I think it's clearer to just call the method `::none()`, so it's obvious from the caller what it is.